src/httpserver.cpp

```diff
@@ -35,6 +35,8 @@
 #endif
 #endif
 
+#include <deque>
```

Comment:
I forgot to factor this out into a separate commit. If memory serves, this is caused by some standard library headers not including others anymore in more recent versions of libc++. It doesn't appear to affect older compilers to add this include in my limited testing.